### DISCUSSION TITLE
[ submit ] button not focusing when deployed.

### DISCUSSION THREAD

**Discussion by tartavull** [charmbracelet/bubbletea#753] [Q&A]
I'm using the example from https://github.com/charmbracelet/bubbletea/blob/master/examples/textinputs/main.go
which works perfectly when I run it on my local machine, but when deployed  the [ submit ] button doesn't focus see `ssh ssh.tartavull.com` 

I'm confused about why the same code behaves differently depending on the host. Any pointers?

Here is the relevant code
```
var (
	focusedStyle        = lipgloss.NewStyle().Foreground(lipgloss.Color("205"))
	blurredStyle        = lipgloss.NewStyle().Foreground(lipgloss.Color("240"))
	cursorStyle         = focusedStyle.Copy()
	noStyle             = lipgloss.NewStyle()
	helpStyle           = blurredStyle.Copy()
	cursorModeHelpStyle = lipgloss.NewStyle().Foreground(lipgloss.Color("244"))
    errorStyle = lipgloss.NewStyle().Foreground(lipgloss.Color("1"))

	focusedButton = focusedStyle.Copy().Render("( Submit )")
	blurredButton = fmt.Sprintf("[ %s ]", blurredStyle.Render("Submit"))
)
```
In the local server the Submit button changes color, but in the remote it only goes from [ Submit] to (submit)

**Comment by meowgorithm** [comment]
I believe this has to do with your terminal and environment. What OS and terminal are you using? And are you using a multiplexer (i.e. tmux, screen, zellij)?

Often the culprit is that `TERM` isn’t being set correctly in your SSH session. As a quick test you can try setting `TERM=xterm-256color ssh ssh.tartavull.com`.

Fwiw, I’m able to see colors in your example just fine.

Also, are you using a custom Lip Gloss renderer in your Wish app? You ideally should; see [example](https://github.com/charmbracelet/lipgloss/blob/master/examples/ssh/main.go).
